appdatabase/migrations/sql/1678275064_add_created_at_to_saved_addresses.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE saved_addresses ADD COLUMN created_at INT DEFAULT 0;
```

Comment:
How about using NULL to mark that a value was not set? Then in select, we can use `coalesce` to substitute `NULL` with a default value of `0`

Reply:
@stefandunca I think that the request was to sort the addresses by `created_at` time, not by the time when they were updated last time.

Reply:
> @stefandunca I think that the request was to sort the addresses by created_at time, not by the time when they were updated last time.

Thanks for the clarification. The implementation looked to me like it is updating the `created_at` timestamp on upsert (update also not only creation)

> @stefandunca Could you please explain why we would need to substitute NULL with 0 in select? The result will be 0 at the end, right? And 0 in database for timestamp actually means that the value has not been set.

It looked to me that `NULL` might say better that the field was not set (before feature). I agree that 0 is also as good and simpler.